v3/lgr_state.hpp

```diff
@@ -73,6 +73,14 @@ class state {
   hpc::time<double> dt = 0.0;
   hpc::time<double> max_stable_dt;
   hpc::dimensionless<double> min_quality;
+
```

Comment:
Here I set a state for temperature, because the `hyper_ep` model's yield surface is temperature dependent.  I thought this was something tracked by lgr in v2?

Reply:
yea v2 has temperature, v3 doesn't on GitHub yet